finalfrontier/src/vec_simd.rs

```diff
@@ -7,58 +7,71 @@ use std::arch::x86::*;
 #[cfg(target_arch = "x86_64")]
 use std::arch::x86_64::*;
 
-/// Dot product: u · v
-///
-/// This SIMD-vectorized function computes the dot product
-/// (BLAS sdot).
 cfg_if! {
     if #[cfg(target_feature = "avx")] {
+        /// Dot product: u · v
+        ///
+        /// This SIMD-vectorized function computes the dot product
+        /// (BLAS sdot).
         pub fn dot(u: ArrayView1<f32>, v: ArrayView1<f32>) -> f32 {
             unsafe { dot_f32x8(u, v) }
         }
     } else if #[cfg(target_feature = "sse")] {
+        /// Dot product: u · v
+        ///
+        /// This SIMD-vectorized function computes the dot product
+        /// (BLAS sdot).
         pub fn dot(u: ArrayView1<f32>, v: ArrayView1<f32>) -> f32 {
             unsafe { dot_f32x4(u, v) }
         }
     } else {
+        /// Unvectorized dot product
```

Comment:
Maybe also change this to:

```
/// Unvectorized dot product: u · v
```

Similarly for scaled addition below.

Reply:
Fixed